core/logger/logger.go

```diff
@@ -177,7 +179,7 @@ func NewLogger() Logger {
 	for _, msg := range parseErrs {
 		l.Error(msg)
 	}
-	return l
+	return l.Named(fmt.Sprintf("%s@%s", static.Version, static.Sha[:7]))
```

Comment:
I wonder if we should make this a helper func? Maybe `logger.VersionName() or something?`

Reply:
+1

Reply:
I extracted only the name formatting to a pkg helper func baseLoggerName